jupyterhub/apihandlers/base.py

```diff
@@ -234,13 +250,26 @@ def user_model(self, user, include_servers=False, include_state=False):
             'pending': None,
             'created': isoformat(user.created),
             'last_activity': isoformat(user.last_activity),
+            'auth_state': '',  # placeholder, filled in later
```

Comment:
Use `None` here instead of empty string, just so it keeps the right type (None or dict, never str).